Detections/SecurityEvent/powershell_empire.yaml

```diff
@@ -36,6 +40,29 @@ query: |
   | where EfectiveCommand matches regex regexEmpire
   | project TimeGenerated, Computer, Account = SubjectUserName, AccountDomain = SubjectDomainName, FileName = Process, EfectiveCommand, decodedCommand, encodedCommand, CommandLine, ParentProcessName
   | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer
+  ),
+  (WindowsEvent
+  | where EventID == 4688 and not(EventData has_any ('gc_worker.exe', 'gc_service.exe'))
+  | where EventData has_any ("-encodedCommand", "powershell.exe","powershell_ise.exe","pwsh.exe")
```

Comment:
KQL may optimze for that, but I would start with the positive tests and than do the exclusion with not.